lib/config_validator.rb

```diff
@@ -2,7 +2,7 @@ class ConfigValidator
   def validate(env = ENV)
     bad_keys = keys_with_bad_values(env, candidate_keys(env))
     return unless bad_keys.any?
-    raise warning(bad_keys).tr("\\\n", ' ')
+    raise warning(bad_keys, (env == ENV ? ' in environment' : '')).tr("\n", ' ')
```

Comment:
`env` will always be `ENV` since that is the default, and we don't set it to anything else. So, instead of this conditional, what do you think about updating the message in the `warning` method to say something like "You have invalid values (yes/no) for [bad keys] in config/application.yml or your environment. Please change them to true or false."